tea.go

```diff
@@ -396,8 +396,7 @@ func (p *Program) Run() (Model, error) {
 		p.renderer = newRenderer(p.output, p.startupOptions.has(withANSICompressor))
 	}
 
-	// Check if output is a TTY before entering raw mode, hiding the cursor and
-	// so on.
+	// Unless the user has specified a custom input, we have ensured that this is a TTY.
```

Comment:
I believe this comment has been stale since 0780601791ff7f9b641c0163d1209faf0cc54c16

Regardless, the original tty check (e84314c622c18316016fe2ab164b5f5679895e94) has been gone for a long time.